python/datamodel/nodesim/datamodel.py

```diff
@@ -62,6 +40,155 @@ def Wpid(self, value):
         self._Wpid = value
 
 @pcc_set
```

Comment:
Is this being pushed to store? If you are just using them for local pcc calculations dont use @pcc_set. just use them directly.